imap_processing/cli.py

```diff
@@ -1016,16 +1016,8 @@ def do_processing(
                 raise ValueError(
                     "Multiple science files processing is not supported for SWE L1B."
                 )
-            l1a_file_path = science_files[0]
-            # read CDF file
-            l1a_dataset = load_cdf(l1a_file_path)
-            # TODO: fix this to read multiple files as needed.
-            # TODO: pass pandas dataframe to swe_l1b
-            # in_flight_cal_path = dependencies.get_file_paths(
-            #     "swe", "l1b-in-flight-cal"
-            # )[0]
-            # TODO: read lookup table and in-flight calibration data here.
-            datasets = swe_l1b(l1a_dataset)
+
+            datasets = swe_l1b_science(dependencies.serialize())
```

Comment:
You might be able to ignore the documentation issues from imap-data-access.* within the nitpick ignore cross references of `conf.py`
```suggestion
            datasets = swe_l1b_science(dependencies)
```

Reply:
Looks like I pushed some changes while you were reviewing. Sorry. This comment has been addressed